### PR TITLE
Feature/firebase config part5/arc 1924

### PR DESCRIPTION
**Pull request by andguladzeshio**
https://spaceneobank.atlassian.net/browse/ARC-1924

### CODE REVIEW

**Review by giorgiqemoklidze — APPROVED** [review]

**Review by GiorgiMamagulashvili16 — APPROVED** [review]

File: app/src/main/java/com/example/taskplanner/model/ProjectTask.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class ProjectTask(
+    val taskId: String = "",
+    val name: String = "",
+    val description: String = "",
+    val startDate: String = "",
+    val endDate: String = "",
+    val progress: String = Progress.TODO
```
Comment (AnaRusadze) [inline app/src/main/java/com/example/taskplanner/model/ProjectTask.kt:14]: change name of "progress" to status, also change enum name to Status

File: app/src/main/java/com/example/taskplanner/util/Progress.kt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package com.example.taskplanner.util
 
-enum class Progress {
-    TODO,
-    IN_PROGRESS,
-    DONE
+object Progress {
```
Comment (AnaRusadze) [inline app/src/main/java/com/example/taskplanner/util/Progress.kt:3]: why did you change enum to object?

**Review by AnaRusadze — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/taskplanner/model/ProjectTask.kt (hunk 1 above)
Comment (andguladzeshio) [reply app/src/main/java/com/example/taskplanner/model/ProjectTask.kt]: done

File: app/src/main/java/com/example/taskplanner/util/Progress.kt (hunk 2 above)
Comment (andguladzeshio) [reply app/src/main/java/com/example/taskplanner/util/Progress.kt]: changed back to enum

**Review by AnaRusadze — APPROVED** [review]

**Pull request merged**